Explorer/Assets/DCL/PluginSystem/Global/MultiplayerMovementPlugin.cs

```diff
@@ -91,6 +91,7 @@ public void InjectToWorld(ref ArchSystemsWorldBuilder<Arch.Core.World> builder,
             RemotePlayerAnimationSystem.InjectToWorld(ref builder, settings.Value.ExtrapolationSettings, settings.Value);
             CleanUpRemoteMotionSystem.InjectToWorld(ref builder);
             MultiplayerMovementDebugSystem.InjectToWorld(ref builder, arguments.PlayerEntity, realmData, debugBuilder, remoteEntities, playerTransform, debugSettings.Value, settings.Value, entityParticipantTable, remoteMetadata);
+            SDKAvatarShapesMotionSystem.InjectToWorld(ref builder);
```

Comment:
it should be in `AvatarShapePlugin`

Reply:
Taking a look with Pravus, we decided to inject the system from `AvatarPlugin` instead of `AvatarShapePlugin`: https://github.com/decentraland/unity-explorer/pull/3297/commits/9635675255e1c41301289e5b7b3bfea394c5f38e